scripts/export_sqlite_to_sql.py

```
@@ -53,6 +53,22 @@ def export_table_data(cursor, table_name, output_file):
         table_name: Name of table to export
         output_file: File object to write INSERT statements to
     """
+    # Get table schema to identify boolean columns
+    cursor.execute(f"PRAGMA table_info({table_name})")
+    table_info = cursor.fetchall()
```

Ruff (0.14.8)

Local variable `table_info` is assigned to but never used

Remove assignment to unused variable `table_info`

(F841)